src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -152,9 +165,9 @@ void SX1280Driver::SetMode(SX1280_RadioOperatingModes_t OPmode)
         break;
 
     case SX1280_MODE_RX:
-        buf[0] = 0x00; // periodBase = 1ms, page 71 datasheet, set to FF for cont RX
-        buf[1] = 0xFF;
-        buf[2] = 0xFF;
+        buf[0] = 0x00; // periodBase = 15.625us, page 71 datasheet
```

Comment:
I know this isn't your code but should be Table 11-24 page 79 (in my datasheet) not page 71

Reply:
Comment and defines added to the top of the file to address this